fab/operations/supervisor.py

```diff
@@ -284,7 +284,7 @@ def restart_all_except_webworkers():
 def _decommission_host(host):
     files.comment(
         '/etc/nginx/sites-available/{}_commcare'.format(env.environment),
-        '^[ ]*server {}'.format(host),
+        '^[ ]*server[ ]+{}'.format(host),
```

Comment:
ha ` +` and `[ ]+` are the same thing, I guess the latter's a little clearer but threw me off for a sec. 👍